bundles/org.openhab.binding.valloxmv/src/main/ressources/ESH-INF/thing/thing-types.xml

```diff
@@ -176,15 +176,14 @@
 
 	<channel-type id="filterchangeddate" advanced="true">
 		<item-type>DateTime</item-type>
-		<label>Last filter change</label>
+		<label>Last Filter Change</label>
 		<description>Date filter was changed last time</description>
-		<category>Date</category>
 		<state readOnly="true" />
 	</channel-type>
 
 	<channel-type id="remainingfilterdays" advanced="true">
-		<item-type>Number</item-type>
-		<label>Days until filter change</label>
+		<item-type>Number:Time</item-type>
```

Comment:
Afaik this requires a code change as well. (It must be a QuantityType now in code, I think)